extensions/wikia/DesignSystem/controllers/DesignSystemGlobalFooterController.class.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class DesignSystemGlobalFooterController extends WikiaController {
+	public function index() {
+		$this->setVal( 'model', $this->getData() );
+	}
+
+	public function section() {
```

Comment:
As implemented, these are all vulnerable to XSS. Most of the values from the API response should be escaped to be safe. Also, if this class doesn't need to be called outside of PHP, extend `WikiaService` instead of `WikiaController` so they can't be called directly via wikia.php.